lms/views/admin.py

```diff
@@ -93,6 +93,12 @@ def update_instance(self):
             self.request.params.get("groups_enabled") == "on",
         )
 
+        ai.settings.set(
+            "blackboard",
+            "files_enabled",
+            self.request.params.get("blackboard_files_enabled") == "on",
```

Comment:
We could call this param "blackboard.files_enabled" or "settings.blackboard.files_enabled" which would pave the way to fairly easily automate adding these if we had more. We don't have to do that now, but it's not much different.